instrumentation/kafka-clients/src/main/java/brave/kafka/clients/TracingConsumer.java

```diff
@@ -49,12 +49,13 @@
     this.remoteServiceName = remoteServiceName;
   }
 
-  @Deprecated @Override public ConsumerRecords<K, V> poll(long timeout) {
-    return poll(Duration.ofMillis(timeout));
+  // Do not use @Override annotation to avoid compatibility issue version < 1.0
```

Comment:
`version < 2.0`
Also can you put this comment everywhere you removed the annotation ? May avoid some mistakes in the future ;)

Reply:
Yeap